test/cpp/jit/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ if(USE_CUDA)
   list(APPEND JIT_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/test/test_gpu_transpose.cpp)
   list(APPEND JIT_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/test/test_gpu_rng.cu)
   list(APPEND JIT_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/test/test_gpu_utils.cpp)
+  list(APPEND JIT_TEST_SRCS ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/test/test_index_select.cpp)
```

Comment:
Could you move the test to `torch/csrc/jit/codegen/cuda/test/test_gpu_indexing_ops.cpp`?
It is a new file added in https://github.com/csarofeen/pytorch/pull/2179/files#diff-ef65e50bd5d7c7c2e6e34f57fa8d01e977252ced3f6e5f77af1cbcdef2cc744a?